nemo/collections/asr/modules/__init__.py

```diff
@@ -12,16 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from nemo.collections.asr.modules.activations import Swish
```

Comment:
Since Swish is a nn.module and not a NeuralModule, it probably should not be in the modules directory.

Reply:
moved.